test/e2e/serial/tests/operator_profile_update.go

```diff
@@ -0,0 +1,282 @@
+/*
```

Comment:
let's think a bit more about the file name. The purpose of the tests is to check when we don't have any NRT data. The fact we leverate the newly added option and we update a profile is an implementation detail, not so important it leaks on the file name